src/content/translations/sl/defi/index.md

```diff
@@ -0,0 +1,324 @@
+
+Pogodbe so prav tako javne, da jih lahko kdorkoli pregleda in revidira. To pomeni, da bodo slabe pogodbe pogosto hitro prišle pod drobnogled skupnosti.
+
+To pomeni, da trenutno obstaja potreba za zaupanje bolj tehnično podkovanim članom Ethereum skupnosti, ki znajo brati programsko kodo. Skupnost, ki temelji na odprtokodnosti pomaga nadzorovati razvijalce, toda ta potreba bo s časom izginila, ko bodo pametne pogodbe postale bolj berljive in bodo razviti drugi načini dokazovanja zanesljivosti programske kode.
+
+## Ethereum in DeFi {#ethereum-and-defi}
+
+Ethereum je popolna podlaga za DeFi iz več razlogov:
+
+- Nihče si ne lasti Ethereuma ali pametnih pogodb, ki živijo na njem – to vsem omogoča priložnost za uporabo DeFi. To prav tako pomeni, da za vas nihče ne more spremeniti pravil.
+- DeFi produkti v ozadju vsi govorijo isti jezik: Ethereum. To pomeni, da veliko produktov brez težav deluje skupaj. Žetone lahko posodite na eni platformi in zamenjate žetone, ki nosijo obresti na drugem trgu v povsem drugačni aplikaciji. To je podobno kot, da bi lahko unovčili točke zvestobe na vaši banki.
+- Žetoni in kriptovalute so vgrajeni v Ethereum, skupno knjigo – beleženje transakcij in lastništva je nekako bistvo Ethereuma.
+- Ethereum dovoljuje popolno finančno svobodo – večina produktov ne bo nikoli hranila vaših sredstev, kar vam prepušča nadzor.
+
+DeFi si lahko predstavljate v plasteh:
+
+1. Blokovna veriga – Ethereum vsebuje zgodovino transakcij in stanje računov.
+2. Sredstva – [ETH](/eth/) in drugi žetoni (valute).
+3. Protokoli – [pametne pogodbe](/glossary/#smart-contract), ki zagotavljajo funkcionalnost, na primer storitev, ki omogoča decentralizirano posojanje sredstev.
+4. [Aplikacije](/dapps/) – produkti, ki jih uporabljamo za upravljanje in dostop do protokolov.
+
+## Razvijajte DeFi {#build-defi}
+
+DeFi je odprtokodno gibanje. DeFi protokoli in aplikacije so vam vse odprte za pregled, oddvojitev in inovacije. Zaradi tega kupa plasti (vse si delijo isto osnovno blokovno verigo in sredstva) se lahko protokoli mešajo in ujemajo z namenom odklepanja edinstvene kombinacije priložnosti.
+
+<ButtonLink to="/developers/docs/dapps/">Več o razvoju dappov</ButtonLink>
+
+## Dodatno branje {#futher-reading}
+
+### DeFi podatki {#defi-data}
+
+- [DeFi Pulse](https://defipulse.com/)
+- [DeFi Prime](https://defiprime.com/)
+
+### DeFi članki {#defi-articles}
+
+- [Kaj so decentralizirane finance?](https://docs.ethhub.io/built-on-ethereum/open-finance/what-is-open-finance/) – _ETHHub, redno posodobljen_
+- [Začetniški vodnik po DeFi](https://blog.coinbase.com/a-beginners-guide-to-decentralized-finance-defi-574c68ff43c4) – _Sid Coelho-Prabhu, 6. januar 2020_
+
+### Videoposnetki {#videos}
+
+- [Finematics - decentralized finance education](https://finematics.com/) – _Videos on DeFi_
```

Comment:
@lukassim any reason we wouldn't have translated this resource but translated the rest?

Reply:
No, this should be translated.
Looking at Crowdin, the translator translated this, but the reviewer changed it back to English.

Reply:
@minimalsm Here's the translation:
[Finematics - izobraževanje na temo decentraliziranih financ](https://finematics.com/) – _Videoposnetki o decentraliziranih financah_